scripts/ci/python_lint.sh

```diff
@@ -0,0 +1,184 @@
+#!/bin/bash
+# This script allows a user to lint a python file or directory of python files
+
+# global variables
+LINT_ALL=""
+LINT_SINGLE=""
+TARGET_MOUNT="/usr/tmp"
+LOG_FILE="lint.log"
+DOCKER_CONTAINER=""
+POPTIONS=""
+
+# public dockerhub image created by a member of the sherlock team
+runLintContainer()
+{
+        echo "Running lint container..."
+        docker run -i -d \
+               --name sherlock_lint \
```

Comment:
Do we need to name it 'sherlock'?  I think that might lead to some confusion for someone without background knowledge.

Reply:
oh no, we don't. I didn't catch that from copying and pasting it from sherlock